src/utils.js

```diff
@@ -86,3 +86,58 @@ export function toDataBody(value) {
 export function qsify(obj) {
   return toQuerystring(JSON.parse(JSON.stringify(obj)));
 }
+
+/**
+ * Checks if a version is within the provided range.
+ *
+ * @param  {String} version The version to check.
+ * @param  {String} min     The minimum version (inclusive).
+ * @param  {String} max     The minimum version (exclusive).
+ * @throws {Error} If the version is outside of the provided range.
+ */
+export function checkVersion(version, min, max) {
```

Comment:
That's your call, but I had to see the examples to understand what these min and max versions were about. Maybe just in the esdoc ?